clouddiscovery/provider/vpc/resource/network.py

```diff
@@ -583,3 +586,62 @@ def get_resources(self) -> List[Resource]:
                         )
 
         return resources_found
+
+
+class RESTAPIPOLICY(ResourceProvider):
+    def __init__(self, vpc_options: VpcOptions):
+        """
+        Rest api gateway policy
+
+        :param vpc_options:
+        """
+        super().__init__()
+        self.vpc_options = vpc_options
+
+    @exception
+    def get_resources(self) -> List[Resource]:
+
+        client = self.vpc_options.client("apigateway")
+
+        resources_found: List[Resource] = []
+
+        # get REST API available
+        response = client.get_rest_apis()
+
+        message_handler("Collecting data from REST API Policies...", "HEADER")
+
+        with ThreadPoolExecutor(15) as executor:
+            results = executor.map(
+                lambda data: self.analyze_restapi(data), response["items"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lambda may not be necessary](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5694158)

Reply:
Ok, makes sense.. I just copied from S3.

Reply:
Check now.